backend-tests/tests/test_useradm_enterprise.py

```diff
@@ -49,8 +49,8 @@ def clean_migrated_mongo(clean_mongo):
     yield clean_mongo
 
 
-@pytest.fixture(scope="function")
-def tenants_users(clean_migrated_mongo):
+@pytest.fixture(scope="function", params=["os", "professional", "enterprise"])
```

Comment:
nit: isn't something in the lines of `@pytest.mark.parametrize("plan", ["os", "professional", "enterprise"])` more elegant?

Reply:
I don't think you can use pytest.mark.parametrize on a fixture, only on a test.